test/integration/verify/controls/aws_config_recorder.rb

```diff
@@ -15,7 +15,8 @@
     its('recorder_name') { should eq aws_configuration_recorder_name }
     its('role_arn') { should eq aws_config_recorder_role_arn }
     its('resource_types') { should eq [] }
-    it { should_not be_recording }
+    its('last_status') { should eq 'FAILURE'}
```

Comment:
Can we pass the FAILURE from the variable like how you passed it in the recorder_name. The reason is if the status is SUCCESS then the integration test will fail.

Reply:
I did checked the terraform resource for recorder_status buts 'No additional attributes are exported.'. Ref:  https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/config_configuration_recorder_status.
Therefore, hardcoding was the only option.